src/CMakeLists.txt

```diff
@@ -214,7 +215,9 @@ set(GAMEENGINE_SRC
     w3d/renderer/missing.cpp
     w3d/renderer/pointgr.cpp
     w3d/renderer/pot.cpp
+    w3d/renderer/predlod.cpp
     w3d/renderer/render2d.cpp
+    w3d/renderer/rendobj.cpp
```

Comment:
when the full word is only a bit longer, shortening feels like yuck, IMO. But then again I come from Windows and I'm biased against the agressive linux-style shortenings.